### ISSUE TITLE
Is this repository still active

### ISSUE THREAD

**Issue by goetzk** [ViaSat/ansible-duounix#3]
Hi,

I'm slowly building up a patchset and trying to decide if I should try upstreaming them here or maintain my own fork.

thanks,
Karl.

**Comment by goetzk** [comment]
Hi,
Wondering if anyone has had a chance to review this query yet.

thanks,
Karl

**Comment by skemper** [comment]
Hi Karl,

If you'd like to submit a PR, please do!  I apologize for the delay in responding to you.  I've been tasked with updating this repo so I'll be looking at the existing PRs, doing some updated testing, and getting the updates published.

**Comment by goetzk** [comment]
Hi @skemper ,
Thanks for getting back to me!
My fork is publicly available - if you'd like a look - and carries quite a few changes. They should be more or less tested as I use this repo in production for a couple of servers (barring any accidental role drift).

https://github.com/Viasat/ansible-duounix/compare/master...goetzk:local-workarounds

I will attempt to open PRs early next week if you would prefer not to import everything as is.

**Comment by skemper** [comment]
It looks like some of the updates I made today are similar to the ones you've done as well.  GitHub won't recognize you as a contributor if you don't send the PR, so I'll let you decide.  If you want the git-log glory, feel free to open a PR with your updates; otherwise, I can manually pull the updates from your fork, and we'll add your name in a Contributors list in the README (which I have to add anyway since we merged in a couple other PRs today).

I won't be back in the office until Thursday, so no rush.
